.github/workflows/documentation-creation.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches: [ master, main, Matrix, Leia ]
 
+permissions:
```

Comment:
Are you sure this is enough? gh-pages action actually pushes/writes to another repository

Reply:
Yes. The permissions are for the `GITHUB_TOKEN` and it is not configured to use that. It is configured to use `secrets.DOC_ACTIONS_DEPLOY_KEY`.

https://github.com/xbmc/xbmc/blob/c8556bda1854e345faed020a5190a3ec0f86145b/.github/workflows/documentation-creation.yml#L47